recipes/starshot_accessibility/recipe.yml

```diff
@@ -0,0 +1,24 @@
+name: Accessibility
+type: Starshot
+description: 'Sets up accessibility tools and functionality.'
+recipes:
+  - starshot_admin_theme
+install:
+  - a11y_form_helpers
```

Comment:
Not sure we should add this. It's only got 242 installs, and it's not clear from the project page what exactly it _does_, beyond a vague "makes forms more accessible". I think this is something we should consider adding separately (i.e., new issue).

Reply:
I was using a11y_form_helpers as it came from the recommendation on this page:
https://www.drupal.org/docs/getting-started/accessibility/contributed-modules-for-extending-accessibility-in-drupal#s-core-improvements
I only actually used it to ensure inline_form_errors was switched on, which I have now included in the starshot recipe itself.